test/functional/legacy/011_autocommands_spec.lua

```diff
@@ -61,7 +62,7 @@ describe('file reading, writing and bufnew and filter autocommands', function()
 
   if not has_gzip() then
     pending('skipped (missing `gzip` utility)', function() end)
-  else
+  elseif not iswin() then
```

Comment:
instead change the condition on line 62 to:

    if iswin() or not has_gzip() then